src/test/java/uk/gov/hmcts/reform/sendletter/encryption/PgpEncryptionUtilTest.java

```diff
@@ -21,14 +21,10 @@ public void should_encrypt_and_create_pgp_encrypted_zip_file_when_valid_public_k
         byte[] pubKey = Resources.toByteArray(getResource("pubkey.asc"));
         PGPPublicKey pgpPublicKey = PgpEncryptionUtil.loadPublicKey(pubKey);
 
-        String fileNamePrefix = "encrypted";
-        String fileNameSuffix = "pgp";
-
         //when
         byte[] pgpEncryptedZip = PgpEncryptionUtil.encryptFile(
             inputZipFile,
-            fileNamePrefix,
-            fileNameSuffix,
+            "encrypted.zip",
```

Comment:
file name is different that the name of the file that we pass. is this how this class is going to be used in real life?

Reply:
you can use any file name in our real service we would be passing something similar to `CMC001_cmcclaimstore_22032018152746_d07c653b-9bcd-4c54-85ca-e22aa321d501.zip`